src/olympia/addons/tests/test_tasks.py

```diff
@@ -249,3 +252,64 @@ def test_add_static_theme_broken_lwt(self):
             licenses.LICENSE_COPYRIGHT_AR.builtin, [rating])
         # Double check its the exact category we want.
         assert static_theme.all_categories == [default_category]
+
+
+@override_settings(ENABLE_ADDON_SIGNING=True)
+class TestMigrateLegacyDictionaryToWebextension(TestCase):
+    def setUp(self):
+        self.user = user_factory(
+            id=settings.TASK_USER_ID, username='taskuser',
+            email='taskuser@mozilla.com')
+        with freeze_time('2017-07-27 07:00'):
+            self.addon = addon_factory(
+                type=amo.ADDON_DICT, target_locale='fr',
+                version_kw={'version': '6.3'})
```

Comment:
After #8068 `addon_factory()` defaults to a random created date (or rather, it was already supposed to be a random date but it was broken before) so freeze_time doesn't do anything.

Reply:
Not the version though, and that's what I cared about (for `current_version` "promotion" to work, I need the older version to be guaranteed to be in the past compared to the new one I'm creating)